docs/manual/guides/local-installation/ddev.de.md

```diff
@@ -80,42 +80,58 @@ mkdir -p  ~/Projekte/contao/contao-ddev && cd ~/Projekte/contao/contao-ddev
 DDEV-Konfiguration anlegen mit:
 
 ```shell
-ddev config
+ddev config --project-type=php --docroot=public --webserver-type=apache-fpm --php-version=8.2 --create-docroot
```

Comment:
Maybe include [timezone](https://ddev.readthedocs.io/en/stable/users/configuration/config/#timezone) here too ("Europe/Berlin")? Maybe with a note, that you can adjust these settings at any time, depending on the project.

Reply:
I have added the parameter Timezone Europe/Berlin